src/Fozzie.java

```diff
@@ -1,11 +1,94 @@
 package fozzie;
 
-import fozzie.CommandFactory;
+import fozzie.fuzzer.CommandFactory;
+
+import java.io.*;
+import java.net.*;
+import java.lang.*;
 
 public class Fozzie {
+	
+	private static final int DRONE_PORT = 5556;
+	private static final int RECEIVE_PORT = 5554;
+	private static final int DRONE_NAVDATA_PORT = 5554;
+	private static final String DRONE_IP_ADDRESS = "192.168.1.1";
+  private static final String SAFE_MODE = "AT*PCMD=1,0,0,0,0,0\r"; // PCMD hovering mode
+
+  public static void main(String[] args) throws Exception {
+    try {
+  	  String cmd = args[0].toUpperCase();
+      int period = Int.parseInt(args[1]);
+      int valid = Int.parseInt(args[2]);
+
+      String commandString = CommandFactory.makeCommand(CommandFactory.CommandType.valueof(cmd), boolean valid!=0).toString();
+      if (valid) {
+        commandString += SAFE_MODE;
+      }
+    }
+
+    catch (Exception e) {
+      if (args.length != 0) {
+        System.out.println(e.getMessage());
+        System.out.println("Generate random command instead.");
+        int period = 50;
+      }
+      String commandString = CommandFactory.makeCommand();
+    }
+  	//setup serverSocket
+  	DatagramSocket serverSocket = new DatagramSocket(RECEIVE_PORT);
```

Comment:
This doesn't work, yeah? might as well remove it and all other code related to it